scalability_and_performance/planning-your-environment-according-to-object-maximums.adoc

```diff
@@ -12,6 +12,11 @@ These guidelines are based on the largest possible cluster. For smaller clusters
 
 In most cases, exceeding these numbers results in lower overall performance. It does not necessarily mean that the cluster will fail.
 
+[NOTE]
+====
+Rapidly changing clusters for example, lots of starting and stopping pods can lower the maximum practical size of a cluster.
```

Comment:
This seems a bit vague to me. I'd suggest something like this:

```adoc
Clusters that experience rapid change, such as those with many starting and stopping pods, can have a lower practical size than documented.
```

I also wonder if this admonition might be better labeled as a warning. What do you think?

Reply:
Changed the note to warning - Addressed